docs/docs/configuration.md

```diff
@@ -0,0 +1,148 @@
+# Configuring Goose
+
+## Adding a toolkit
+To make a toolkit available to Goose, add it to your project's pyproject.toml. For example in the Goose pyproject.toml file:
+```
```

Comment:
maybe breaking this up into two sections: Adding existing toolkits to goose (which would resume from the profile below). Moving the setup in the pyproject.toml to toolkits.md